src/routes/store/games/+page.svelte

```diff
@@ -2,35 +2,39 @@
   import SearchBar from '$partials/games/SearchBar.svelte';
 	import { favoriteIcon, maximiseIcon } from '$lib/Components/iconPaths';
 	import FeaturedCategories from '$partials/games/FeaturedCategories.svelte';
+
+  export let data;
+
+  $: ({games} = data);
 </script>
 
 <div class="container py-28 lg:py-40">
   <div class="row">
     <main class="grid lg:grid-cols-4 grid-cols-2 gap-3 mb-8 px-2">
-      <FeaturedCategories />
+      <FeaturedCategories category={games.category}  />
 
       <SearchBar />
 
-      {#each [1,2,3,4,5,6,7,8,9,10] as n}
+      {#each games.list as item, index}
         <div class="group overflow-hidden relative flex flex-col bg-white shadow rounded-xl dark:bg-neutral-700/30 dark:border-neutral-700 dark:shadow-neutral-700/70 transition hover:drop-shadow-xl hover:shadow-sm hover:-translate-y-1 hover:translate-x-1 hover:scale-[1.025]">
 
-          {#if n%2 == 0}
-            <enhanced:img class="w-full h-auto rounded-xl" src="$lib/images/games-6.avif?aspect=400:320&fit=cover" alt="hero-img-thumb" />
-          {:else if n%3 == 0}
-            <enhanced:img class="w-full h-auto rounded-xl" src="$lib/images/games-5.avif?aspect=400:320&fit=cover" alt="hero-img-thumb" />
+          {#if index%2 == 0}
```

Comment:
This if statement here, what exactly is its purpose?

As far as I can tell, the images in the if blocks are all the same.

Reply:
i thought you had a reason for that, that was why i left it

Reply:
Ok.

I was using it to mock different images before. But since we are using dynamic images now, there is no need anymore